demo/app/main-view-model-android.ts

```diff
@@ -0,0 +1,42 @@
+import {Observable} from "data/observable";
+var loadingIndicator = require("nativescript-loading-indicator");
+
+export class LoadingAndroidTest extends Observable {
+  
+  public showLoader() {
+    loadingIndicator.show({ message: 'Loading test...' });
```

Comment:
Sure. I will mention though with TypeScript I believe single quotes are preferred, and a common convention is also using backticks in implementation files so whenever the need arises to interpolate, you can:
```
let opinion = 'awesome';
`this is ${opinion}`;
```